main-app/app/scripts/bingoNumbers/callingNumbers.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.LongPollingService')
+        .service('longPolling',['$timeout', 'proxy', 'userModel', 'numberService', function ($timeout, proxy, userModel, numberService) {
```

Comment:
1. CamelCase the service names.
2. This doesn't long poll, so don't call it that.